.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ commands:
     steps:
       - run:
           name: Ensure cf cli is installed, otherwise install it.
-          command: ./scripts/cf-check.sh
+          command: sudo ./scripts/cf-check.sh
```

Comment:
We need to call this script with `sudo` when using it on a `machine-executor` job or a permissions error will be raised.